bazel/README.md

```diff
@@ -270,6 +270,16 @@ for how to update or override dependencies.
 1. `bazel build envoy` from the Envoy source directory. Add `-c opt` for an optimized release build or
    `-c dbg` for an unoptimized, fully instrumented debugging build.
 
+## Running Envoy via CLI
+
+After Envoy is built, it can be executed via CLI.
+
+For example, if Envoy was built using the `bazel build -c opt //source/exe:envoy-static` command, then it can be executed from the project's root directory by running:
+
+```
+bazel-bin/source/exe/envoy-static --config-path /path/to/your/envoy/config.yaml
```

Comment:
bazels ability to provide info about where it just built things is limited (cf https://github.com/bazelbuild/bazel/issues/8739) 

as a ~workaround it is possible to use `bazel info` - i believe something like the following should work (and would also hopefully address above comments)

```console

$ $(bazel info bazel-genfiles)/source/exe/envoy-static --config-path /path/to/your/envoy/config.yaml

```